src/object/obj_rpc.h

```diff
@@ -74,9 +65,8 @@
 		0, &CQF_obj_punch,					\
 		ds_obj_punch_handler, NULL, "akey_punch")		\
 	X(DAOS_OBJ_RPC_QUERY_KEY,					\
-	  0, ver == 0 ? &CQF_obj_query_key_0 : &CQF_obj_query_key_1,	\
-	  ver == 0 ? ds_obj_query_key_handler_0 :			\
-	  ds_obj_query_key_handler_1, NULL, "key_query")		\
+	  	0, &CQF_obj_query_key,					\
+	  	ds_obj_query_key_handler, NULL, "key_query")		\
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs